apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.test.js

```diff
@@ -0,0 +1,650 @@
+        expect(foundRight).toHaveProperty('accessRightSet.canReadContacts', false)
+        expect(foundRight).toHaveProperty('accessRightSet.canManageContacts', false)
+    })
+
+    test('Organization', async () => {
+        const [organization] = await registerNewOrganization(user)
+
+        const [app] = await createTestB2BApp(admin)
+        await createTestB2BAppContext(admin, app, organization, { status: 'Finished' })
+        const [right] = await createTestB2BAppAccessRight(admin, serviceUser, app)
+
+        // B2BApp without permissions
+        const countWithoutPermissions = await Organization.count(serviceUserClient, {})
+        expect(countWithoutPermissions).toBe(0)
+        const propertyWithoutPermissions = await Organization.getOne(serviceUserClient, { id: organization.id })
+        expect(propertyWithoutPermissions).toBeUndefined()
+
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await createTestOrganization(serviceUserClient)
+        })
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await updateTestOrganization(serviceUserClient, organization.id, {})
+        })
+        await expectToThrowAccessDeniedErrorToObj(async () => {
+            await Organization.softDelete(serviceUserClient, organization.id)
+        })
+
+        const countByUser = await Organization.count(user, {})
+        expect(countByUser).toBe(1)
+
+        // you cannot create a field "canManageOrganizations: true". Only reading! Always false!
+        await catchErrorFrom(async () => {
+            await createTestB2BAppAccessRightSet(admin, right.id, {
+                canManageOrganizations: false,
+            })
+        }, (e) => {
+            expect(e.errors[0].message).toContain('Field "canManageOrganizations" is not defined by type "B2BAppAccessRightCreateInput"')
+        })
+
+        // add permissions for B2BApp
+        const [accessRightSet] = await createTestB2BAppAccessRightSet(admin, app, {
+            canReadOrganizations: true,
+        })
+        await updateTestB2BAppAccessRight(admin, right.id, { accessRightSet: { connect: { id: accessRightSet.id } } })
+
+        // you can update 'canReadOrganizations'
+        await updateTestB2BAppAccessRightSet(admin, accessRightSet.id, {
+            canReadOrganizations: true,
+        })
+
+        // you cannot update a field "canManageOrganizations". Only reading! Always false!
+        await catchErrorFrom(async () => {
+            await updateTestB2BAppAccessRightSet(admin, accessRightSet.id, {
+                canManageOrganizations: false,
+            })
+        }, (e) => {
+            expect(e.errors[0].message).toContain('Field "canManageOrganizations" is not defined by type "B2BAppAccessRightUpdateInput"')
+        })
+
+        // B2BApp with permission "canReadOrganization"
+        const countWithPermissions = await Organization.count(serviceUserClient, {})
+        expect(countWithPermissions).toBe(1)
```

Comment:
It's cool to check that only one organization is available and no more, but add a check which exactly org is expected to be seen